src/main/java/uk/gov/indexer/monitoring/CloudwatchRecordsProcessedUpdater.java

```diff
@@ -21,7 +21,7 @@ public CloudwatchRecordsProcessedUpdater(String environment, String register) {
         acw.setEndpoint("https://monitoring.eu-west-1.amazonaws.com");
     }
 
-    public void update(long recordsProcessed) {
+    public void update(int recordsProcessed) {
```

Comment:
Caller of this function earlier had long value but after refactoring caller got int value.